tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -79,13 +79,17 @@ public function test_wp_enqueue_script() {
 
 		wp_enqueue_script( 'no-deps-no-version', 'example.com', array() );
 		wp_enqueue_script( 'empty-deps-no-version', 'example.com' );
-		wp_enqueue_script( 'empty-deps-version', 'example.com', array(), 1.2 );
+		wp_enqueue_script( 'empty-deps-version', 'example.com', array(), '1.2' );
 		wp_enqueue_script( 'empty-deps-null-version', 'example.com', array(), null );
+		wp_enqueue_script( 'empty-deps-arg-in-handle-with-ver?arg1=foo&arg2=bar', 'https://example.com/test.js', array(), '2.0' );
+		wp_enqueue_script( 'empty-deps-arg-in-handle-without-ver?arg1=foo&arg2=bar', 'https://example.com/test.js', array(), null );
```

Comment:
Add a test for the following, the expected output should include the handle's query string in the enqueue statement.

```php
wp_register_script( 'registered-no-qs-handle-null-version-enqueued-with-qs', ... );
wp_enqueue_script( 'registered-no-qs-handle-null-version-enqueued-with-qs?arg1=foo&arg2=bar' );
```

:1234: And same below for styles.